experiment/kind-conformance-image-e2e.sh

```diff
@@ -25,7 +25,7 @@ set -o xtrace
 # - get some fixes related to Kubernetes's build changing
 # - don't get surprised when kind changes between now and the next stable release
 # We should switch back to a release tag at the next release.
-STABLE_KIND_VERSION=v0.5.1
+STABLE_KIND_VERSION=v0.6.0
```

Comment:
we should switch kubeconfig management like https://github.com/kubernetes/test-infra/pull/15247

Reply:
Done